app/main.ipynb

```diff
@@ -54,58 +91,57 @@
     "Example:\n",
     "\n",
     "![pie chart](../demo/pie.png)"
-   ],
+   ]
+  },
+  {
    "metadata": {
     "collapsed": false,
     "pycharm": {
-     "name": "#%% md\n"
+     "name": "#%%\n"
     }
-   }
-  },
-  {
+   },
    "cell_type": "code",
-   "execution_count": null,
+   "source": "order_data.groupby(\"name\")[\"quantity\"].sum().sort_values(ascending=False).head(10).plot.pie(y=\"Quantity\", autopct=\"%1.1f%%\", title=\"Top 10 most popular products sold by Quantity\", figsize=(8, 8), startangle=90, pctdistance=0.8)",
```

Comment:
In the pie chart generation for Task 1, the `ylabel` parameter is missing in the `plot.pie()` function call. This might cause an error or incorrect labeling. Consider adding `ylabel=''` to ensure the chart is labeled correctly.